src/main/java/spireMapOverhaul/zones/thefog/events/StandingStones.java

```diff
@@ -0,0 +1,88 @@
+package spireMapOverhaul.zones.thefog.events;
+
+import com.badlogic.gdx.math.MathUtils;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.events.AbstractImageEvent;
+import com.megacrit.cardcrawl.localization.EventStrings;
+import com.megacrit.cardcrawl.rooms.AbstractRoom;
+import spireMapOverhaul.rewards.AnyColorCardReward;
+import spireMapOverhaul.zones.thefog.relics.StoneFragment;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+import static spireMapOverhaul.SpireAnniversary6Mod.makeImagePath;
+import static spireMapOverhaul.util.Wiz.adp;
+
+public class StandingStones extends AbstractImageEvent {
+    public static final String ID = makeID("StandingStones");
+
+    private static final EventStrings eventStrings = CardCrawlGame.languagePack.getEventString(ID);
+
+    public static final String NAME = eventStrings.NAME;
+    public static final String[] DESCRIPTIONS = eventStrings.DESCRIPTIONS;
+    public static final String[] OPTIONS = eventStrings.OPTIONS;
+
+    private int screenNum = 0;
+
+    private static final float HP_HEAL_PERCENT = 0.33F;
+    private static final float A15_HP_HEAL_PERCENT = 0.2F;
+    private static final float HP_DAMAGE_PERCENT = 0.15F;
+    private static final float A15_HP_DAMAGE_PERCENT = 0.2F;
+
+    private final int healAmt;
+    private final int damageAmt;
+
+    public StandingStones() {
+        super(NAME, DESCRIPTIONS[0], makeImagePath("events/TheFog/StandingStones.png"));
+        this.noCardsInRewards = true;
+
+        if (AbstractDungeon.ascensionLevel >= 15) {
```

Comment:
Let's round all this down instead.

The base game rounds damage down in both fights and events, and while the healing bonus from Magic Flower is rounded up in fights, healing in events is mostly rounded down. Library uses `MathUtils.round` like this, but contrast that with Big Fish, Cleric, and Mushrooms, which all round down.

So while there's an argument you could make for rounding up the healing, I think the sanest thing to do is round everything down. That's most consistent with other events for both the damage and the healing, and it keeps this event internally consistent (I wouldn't want to have multiple rounding approaches in the same event).

Really just need to replace the round calls with `(int)`, e.g.:
```
this.healAmt = (int)(adp().maxHealth * A15_HP_HEAL_PERCENT);
this.damageAmt = (int)(adp().maxHealth * A15_HP_DAMAGE_PERCENT);
```
(and so on for the else clause)